src/animationBuilder.tsx

```diff
@@ -25,12 +26,52 @@ const mockTargetValues: LayoutAnimationsValues = {
   currentBorderRadius: 0,
 };
 
+function maybeReportOverwrittenProperties(
+  layoutAnimationStyle: StyleProps,
+  style: NestedArray<StyleProps>,
+  displayName: string
+) {
+  function getCommonProperties(
+    layoutStyle: object,
+    secondStyle: object | Array<object>
+  ) {
+    const secondStyleFlat = Array.isArray(secondStyle)
+      ? secondStyle.flat()
+      : [secondStyle];
+    const commonKeys: Array<string> = [];
+
+    secondStyleFlat.forEach((s) => {
+      if ('initial' in s) {
+        s = s.initial.value;
+      }
+
+      const commonStyleKeys = Object.keys(s).filter((key) =>
+        Object.prototype.hasOwnProperty.call(layoutStyle, key)
+      );
+      commonKeys.push(...commonStyleKeys);
```

Comment:
Why not just use `commonKeys.concat`?

Reply:
Good catch, It seems, that concat should be a little faster: https://www.measurethat.net/Benchmarks/Show/7699/0/array-concat-vs-spread-vs-push-spread-loop-apply#latest_results_block